src/test/java/secretclosetapp/utils/Constants.java

```diff
@@ -0,0 +1,6 @@
+package secretclosetapp.utils;
+
+public class Constants {
```

Comment:
it's better to avoid such abstract class names, for example to change it to StringFormatConstants or smth like this